pnpm-lock.yaml

```diff
@@ -802,6 +803,7 @@ overrides:
   path-to-regexp@<0.1.12: ^0.1.12
   path-to-regexp@>=4.0.0 <6.3.0: '>=6.3.0'
   path-to-regexp@>=7.0.0 <8.0.0: '>=8.0.0'
+  postman-request>qs: ^6.14.1
```

Comment:
The `postman-request` and `@cypress/request` packages are both forks of `request`, which has used tilde specs for its dependencies. I looked through git blame and it's used tilde dependencies for most of its dependencies for many years. I don't think `postman-request` or `@cypress/request` actually need the specific version of `qs` it's looking for.